chapter_appendix-mathematics-for-deep-learning/naive-bayes.md

```diff
@@ -87,7 +96,7 @@ image.shape, label
 ```{.python .input}
 #@tab tensorflow
 image, label = train_images[2], train_labels[2]
-image.shape, label
+image.shape, label.numpy()
```

Comment:
Why is this needed?

Reply:
`tf.keras.datasets` gives `numpy.ndarray`. But at the beginning, I have converted `numpy.ndarray` to tf.Tensor. So if I don't use `label.numpy` answer would contain `shape`, `dtype` etc. To to just get the value of the label, I have used `label.numpy()`.